### PR TITLE
ECOC cookbook update

### PR DESCRIPTION
**Pull request by OXPHOS**

### CODE REVIEW

File: doc/cookbook/source/examples/classifier/multiclass_ecoc_random.rst [hunk 1]
```diff
@@ -13,9 +13,22 @@ There are multiple methods to encode or decode a codebook. In this example, we s
...
-In this examples, we use :sgclass:`CECOCRandomDenseEncoder` as encoder. We also provides :sgclass:`CECOCRandomSparseEncoder` for generating sparse codebooks.
+In this examples, we use :sgclass:`CECOCRandomDenseEncoder` as encoder. :sgclass:`CECOCRandomSparseEncoder` can be applied for generating sparse codebooks.
 
-See :cite:`escalera2010ecoc` for a detailed introduction
+Shogun also provides:
+
+One-vs-Rest Encoder :sgclass:`CECOCOVREncoder`,
+One-vs-One Encoder :sgclass:`CECOCOVOEncoder`,
+Forest ECOC Encoder :sgclass:`CECOCForestEncoder` and
+Discriminant ECOC Encoder :sgclass:`CECOCDiscriminantEncoder`.
```
Comment (karlnapf) [inline doc/cookbook/source/examples/classifier/multiclass_ecoc_random.rst]: I think you just want to reference the base class here. Not all the implementations

**Comment by karlnapf** [comment]
Please squash and we can merge

**Comment by OXPHOS** [comment]
@karlnapf ready

**Comment by karlnapf** [comment]
Thanks!

**Pull request merged**